From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Tue, 24 Apr 2018 14:17:16 -0400
Subject: [PATCH] Revert "shell: make the .desktop file pass "--overview""

This reverts commit 94aeba51e311ce03a3f6cd5e78d8cb45f887267f.

We actually do want to see the last panel opened
---
 shell/gnome-control-center.desktop.in.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/shell/gnome-control-center.desktop.in.in b/shell/gnome-control-center.desktop.in.in
index 4309b7138..d6886a438 100644
--- a/shell/gnome-control-center.desktop.in.in
+++ b/shell/gnome-control-center.desktop.in.in
@@ -2,7 +2,7 @@
 Name=Settings
 # Translators: Do NOT translate or transliterate this text (this is an icon file name)!
 Icon=gnome-control-center
-Exec=gnome-control-center --overview
+Exec=gnome-control-center
 Terminal=false
 Type=Application
 StartupNotify=true
